pkg/api/validation/coverage_test.go

```diff
@@ -28,7 +29,9 @@ var KnownValidationExceptions = []reflect.Type{
 var MissingValidationExceptions = []reflect.Type{
 	reflect.TypeOf(&buildapi.BuildLogOptions{}), // TODO, looks like this one should have validation
 	reflect.TypeOf(&buildapi.BuildLog{}),        // TODO, I have no idea what this is doing
-	reflect.TypeOf(&imageapi.DockerImage{}),     // TODO, I think this type is ok to skip validation (internal), but needs review
+	reflect.TypeOf(&deployapi.DeploymentLogOptions{}),
+	reflect.TypeOf(&deployapi.DeploymentLog{}),
```

Comment:
Can you explain the "only returned, never accepted"? This is a virtual subresource, essentially unused, its client interface is masking a call to "deploymentConfigs/log".